### PR TITLE
I've completed the tasks

### PR DESCRIPTION
**Pull request by SmallPeenwow**
A prompt text will display when the crosshair hover overs an game object and once E is keydown it will destroy the game object. Script is called CollectItem that destroys the game object.

### CODE REVIEW

File: paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        if(Physics.Raycast(ray, out hitInfo, distance, layerMask))
+        {
+            if(hitInfo.collider.GetComponent<Interactable>() != null)
+            {
+                Interactable interactable = hitInfo.collider.GetComponent<Interactable>();
+
+                playerUI.UpdateText(interactable.promptMessage);
+
+                if (Input.GetKeyDown(KeyCode.E))
```
Comment (itbeginshere) [inline paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs:42]: Please make use of the new Input Management System that unity provides. This is the old way of doing things. The video that you watched specifies this

File: paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (itbeginshere) [inline paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs:2]: Please remove unused imports

**Comment by SmallPeenwow** [comment]
I only got a notification now on my email. I'm on it now sorry

File: paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs (hunk 1 above)
Comment (SmallPeenwow) [reply paranomal-game/Assets/Scripts/Player/PlayerInteractable.cs:42]: Have added the new Input Management System and done Scripts for it

**Comment by itbeginshere** [comment]
In future remove any method that you do not use like this Update method. It is fine for now
image

**Pull request merged**